src/components/duration.tsx

```diff
@@ -1,20 +1,7 @@
-import * as moment from 'moment';
 import * as React from 'react';
 
-export const Duration = (props: {durationMs: number, allowNewLines?: boolean}) => {
-    const momentTimeStart = moment.utc(0);
-    const momentTime = moment.utc(props.durationMs * 1000);
-    const duration = moment.duration(momentTime.diff(momentTimeStart));
-    let formattedTime = '';
+import { formatDuration } from '../../v2';
 
-    if (momentTime.diff(momentTimeStart, 'hours') === 0) {
-        formattedTime = ('0' + duration.minutes()).slice(-2) + ':' + ('0' + duration.seconds()).slice(-2) + ' min';
-    } else {
-        if (momentTime.diff(momentTimeStart, 'days') > 0) {
-            formattedTime += momentTime.diff(momentTimeStart, 'days') + ' days' + (props.allowNewLines ? '<br>' : ' ');
-        }
-
-        formattedTime += ('0' + duration.hours()).slice(-2) + ':' + ('0' + duration.minutes()).slice(-2) + ' hours';
-    }
-    return <span dangerouslySetInnerHTML={{__html: formattedTime}}/>;
-};
+export function Duration(props: {durationMs: number}) {
+    return <span>{formatDuration(props.durationMs * 1000)}</span>;
```

Comment:
I must've done the math wrong here or something

Reply:
Oh. `formatDuration` takes `seconds` -- this should be divided by 1000, not multiplied. I must've taken this from the top `momentTime` variable or something

```suggestion
    return <span>{formatDuration(props.durationMs / 1000)}</span>;
```

`formatDuration` handles rounding, so this should be fine